dispatcher.py

```diff
@@ -222,6 +312,8 @@ def main(args):
     parser.add_argument('-a', '--address', type=str, default='127.0.0.1', help='node address')
     parser.add_argument('-p', '--port', type=int, default=4142, help='connection port')
     parser.add_argument('-l', '--level', type=str, default='DEBUG', help='log level')
+    parser.add_argument('-d', '--depth', type=int, default=1, help='recursively downloads depth')
```

Comment:
i understand the help, but is not clear, a  "number of depth desired of recursively downloads"
i dont know, people usually dont know what recursively means

Reply:
It is the same, yours has the words `number` and `desired`, but it is still the same, it is also a project for people who are sure to have an idea of ​​what it means `recursively`. But if you think it necessary feel free to change it